js/codes.js

```diff
@@ -0,0 +1,248 @@
+const codesList = {
```

Comment:
Are these [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) codes?

Where did this list come from? Does it have a license?

I would name this something more descriptive like: `countryNameBy2LetterCode`.

I would also put it inside a `myL3` namespace as above in the comment on `apiKeysTemplate.js`.

Reply:
A very good discussion on this link: https://gist.github.com/keeguon/2310008 
Many people shared the data and picked on of them.

Following is the exact link to JSON file mentioned by @manuelstofer in that discussion:
https://github.com/tarraq/JSON-data-arrays/blob/master/countries/english/countries-key-value.json

Reply:
I am working on suggested changes. Thanks @simonbates